### ISSUE TITLE
two random seeds

### ISSUE THREAD

**Issue by rjschmitt** [schmittkl/FARM#5]
Currently, FARM does use one seed for initialize the network as well as sorting/splitting the data.

[This work](https://arxiv.org/abs/2002.06305) suggests to split these random seeds and to see them as hyperparameter. In order to implement this feature into FARM, there are following code lines to edit:

- If no dev set is defined, a random split will be done. There is no random-value currently used in the function https://github.com/schmittkl/FARM/blob/7305a17979b0a80dbe2dbebe5815450883f20627/farm/data_handler/data_silo.py#L398
- the same is for this line: https://github.com/schmittkl/FARM/blob/7305a17979b0a80dbe2dbebe5815450883f20627/farm/data_handler/data_silo.py#L412

**Comment by schmittkl** [comment]
I tackled that problem in [this branch](https://github.com/schmittkl/FARM/tree/random_seeds_dev_split). A config for testing can be found [here](https://github.com/schmittkl/Trapdoor/blob/main/MS1_FARM_features/correct_random_seeds_gnad_config.json).
Both values are loaded directly into the data silo. I hope this solves the (correct) problem this time. ;)

**Comment by rjschmitt** [comment]
Was this ever added to master?

**Comment by schmittkl** [comment]
> Was this ever added to master?

Yes, it should have been added a while ago: [Pull Request #8](https://github.com/schmittkl/FARM/pull/8)
